core/testdata/testspecs/v2_specs.go

```diff
@@ -218,6 +220,10 @@ func GenerateVRFSpec(params VRFSpecParams) VRFSpec {
 	if params.MinIncomingConfirmations != 0 {
 		confirmations = params.MinIncomingConfirmations
 	}
+	requestedConfsDelay := 0
+	if params.RequestedConfsDelay != 0 {
+		requestedConfsDelay = params.RequestedConfsDelay
+	}
```

Comment:
this could just be `requestedConfsDelay := params.RequestedConfsDelay` without an `if`